crates/misc/rust/src/lib.rs

```diff
@@ -6,6 +6,7 @@ pub mod __rt {
     pub use anyhow;
     pub use cranelift_codegen;
     pub use cranelift_native;
+    pub use wasmtime_api;
     pub use wasmtime_interface_types;
     pub use wasmtime_jit;
```

Comment:
Can all other dependencies other than `wasmtime_api` and maybe the interface types one be removed from this now?

Reply:
(both from here and `Cargo.toml`)

Reply:
> Can all other dependencies other than `wasmtime_api` and maybe the interface types one be removed from this now?

Yes, the intent to remove wasmtime_jit/wasmtime_runtime like dependencies. `wasmtime_interface_types` not yet

Reply:
Ah ok, did you want that to happen for this PR or to happen in a later PR?